src/List/List.component.js

```diff
@@ -508,6 +508,7 @@ const List = React.createClass({
                         <ImportExport
                             d2={d2}
                             columns={settings.get("visibleTableColumns")}
+                            allColumns={allColumns}
```

Comment:
This new prop was missing in PropTypes, added: 30701d3. Fortunately, we don't need this for TS apps.